.changeset/shy-otters-hug.md

```diff
@@ -0,0 +1,10 @@
+---
+'@primer/react': patch
+---
+
+ButtonGroup stories:
+
+- Remove ButtonGroup from `deprecated` stories
+- Add ButtonGroup stories default
+- Add the new storybook link to the documentation
+- Removed broken migration link in Button
```

Comment:
Hi @josepmartins, as these notes go into release notes + changelog, do you think we should skip because they're not super useful to folks that would consume it?

Reply:
Good point. Lets skip it 👍